pkg/config/process.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	"os"
+	"time"
+)
+
+const (
+	// DefaultGRPCConnectionTimeoutSecs sets the default value for timeout when connecting to the agent
+	DefaultGRPCConnectionTimeoutSecs = 60
+)
+
+func setupProcesses(config Config) {
+	procBindEnvAndSetDefault := func(key string, val interface{}, env string) {
+		config.BindEnvAndSetDefault(key, val, "DD_PROCESS_CONFIG_"+env, "DD_PROCESS_AGENT_"+env)
+	}
+	// Note that `BindEnvAndSetDefault` automatically creates environment variables with the `DD_PROCESS_CONFIG` prefix.
```

Comment:
is this comment still valid? `BindEnvAndSetDefault` will not automatically prefix the env with `DD` and we need to specify both of them with `procBindEnvAndSetDefault`.

Reply:
Oh thanks for noticing that! I'll go ahead and remove it